tests/docker-compose.yml

```diff
@@ -45,4 +45,10 @@ services:
             - "monerod"
         ports:
             - 18084:18083
+    monero-wallet-rpc-3:
+        image: ghcr.io/farcaster-project/containers/monero-wallet-rpc:0.17.2.3
+        environment:
+            MONEROD_ADDRESS: monerod:18081
+        ports:
```

Comment:
We can also use `depends_on` here, it just tell docker if monerod cannot start don't start the rest, but it doesn't wait for monerod to be "ready", so this is just a minor thing but we have it elsewhere

Reply:
Done in 9ed75c7563406834645aceac8a064c33b37fc9ee